libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -340,10 +339,10 @@ hiddenPermissionsFromPermissions =
             Set.fromList [ViewSameTeamEmails]
         roleHiddenPerms RoleExternalPartner =
           Set.fromList
-            [ ViewLegalHoldTeamSettings,
+            [ ViewTeamFeature TeamFeatureLegalHold,
+              ViewTeamFeature TeamFeatureSSO,
+              ViewTeamFeature TeamFeatureSearchVisibility,
```

Comment:
How likely is it that we will remember updating this list when we add another feature flag? How can we increase it?

Reply:
I tried this: 76a2ab8

Which had the nice side-effect that I refactored hidden permissions a little: c9f835c